sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -109,7 +109,7 @@ trait ExpressionEvalHelper {
     }
 
     val actual = plan(inputRow)
-    val expectedRow = new GenericRow(Array[Any](CatalystTypeConverters.convertToCatalyst(expected)))
+    val expectedRow = InternalRow.fromSeq(Array(CatalystTypeConverters.convertToCatalyst(expected)))
```

Comment:
Sound reasonable, it's anonying to have so many `UTF8String.fromString` in test cases.